recipes/color-operations/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "color-operations" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/color-operations-{{ version }}.tar.gz
+  sha256: 97a0fd38e65eb255d81c4eafbf1cc46fe47d9744fcd35d23c5524ca951a0c357
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.8
+    - numpy
+    - cython
+    - pip
+  run:
+    - python >=3.8
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - color_operations
```

you might need to explicitly list the modules 
```suggestion
    - color_operations.colorspace
```

---

I just noticed that they do that in the dask feedstock :shrug:

---

Nah that is probably not useful